### DISCUSSION TITLE
Enable option to remove OpenRCT2 Official void land textures (both surface and wall) from giant screenshots

### DISCUSSION THREAD

**Discussion by Xtreme97-RCT** [OpenRCT2/OpenRCT2#17874] [Feature requests]
This would allow giant screenshot to be made where the void land textures are treated the same as the outer map void and made transparent in the final giant screenshot. Currently maps that use void land texture as a "blacktile" effect look like this when a giant screenshot is taken:
![Xtreme97 Workbench 2021 2022-08-21 17-33-51](https://user-images.githubusercontent.com/64152940/185801391-61edc38e-8ac6-46be-93fd-71ba1d349c3d.png)

This would make aerial making a lot easier for parks that use void texture to shape park maps.

**Comment by IntelOrca** [comment]
How safe is it to make this colour index transparent?

**Reply by Xtreme97-RCT** [reply]
It's the darkest shade of the black-white group so making all instances of it transparent would render a number of pixels inside the park map also transparent unfortunately.

**Reply by Xtreme97-RCT** [reply]
Just to demonstrate in GIMP, I've selected the single black colour here and it highlights a lot of the B&M track as well:
![image](https://user-images.githubusercontent.com/64152940/185803758-419d92bc-b528-4ad1-ad9a-bed08d1a3247.png)

**Comment by frutiemax** [comment]
I believe that a post effect compositing algorithm could eliminate that color index starting from the bottom middle pixel that are connected in islands.

**Comment by frutiemax** [comment]
Or simpler, eliminate islands of that color index that have more than X pixels connected

**Reply by spacek531** [reply]
That is certain to not work 
![image](https://user-images.githubusercontent.com/3454156/186045191-38fba2e3-ad17-41b5-b8e3-9a4b71b2b2a8.png)

**Reply by frutiemax** [reply]
You can increase the number of pixels required for the island.

**Reply by spacek531** [reply]
Then it will leave the background on places where it should not be

**Comment by spacek531** [comment]
In my opinion, a flag in the object that paints objects a special color (ala ghost behavior) that is opaque but is exported as transparent is an avenue to pursue.
